### ISSUE TITLE
May not work with VS 2017 RTM

### ISSUE THREAD

**Issue by gehnster** [aspnet/JavaScriptServices#745]
`The program '[26784] dotnet.exe' has exited with code -2147450749 (0x80008083).
The program '[30640] chrome.exe: WebKit' has exited with code -1 (0xffffffff).
The program '[26836] iisexpress.exe' has exited with code 0 (0x0).`

Getting the following error when creating a new project with 'yo aspnetcore-spa' Creating an React-Redux .csproj project.

**Comment by SteveSandersonMS** [comment]
Can you provide repro steps that lead to this error on other machines?

I just created a new React-Redux app using `yo aspnetcore-spa`, then opened and ran it from VS 2017 RTW, without any trouble.

**Comment by DanHarman** [comment]
I can't repro this either.

**Comment by jamessantiago** [comment]
I'm currently getting this issue too after installing visual studio 2017 RTM.  I get this error for both my existing asp.net core project and new console and web projects created in 2017.  So my repro would be new .net core project -> run/debug.  I'd imagine this means there's an issue with my system such as having the wrong sdk version or something.  I believe I correctly updated to the latest .net core SDK and runtime.  The SDK for these projects are set to Microsoft.NETCore.App 1.1.1.  Running dotnet return version 1.1.0 and build 928f77c4bc3f49d892459992fb6e1d5542cb5e86.  Running dotnet --info returns 1.0.0-preview2-1-003177.

**Comment by grufffta** [comment]
@jamessantiago  you need to add/update your global json - preview2 is project.json
**global.json**
`{
  "sdk": { "version": "1.0.0" }
}
`

**Comment by jamessantiago** [comment]
Ok, I got it working.  I think what was happening was that the x86 path for dotnet was being used and I had old versions of dotnet installed there.  To fix this I went through and ran uninstall against every dotnet sdk/runtime installer I downloaded (no easier way?) then reinstalled the newest x64 sdk and runtime.  I saw that the path of dotnet changed from x86 to x64 after I did this (see `get-command dotnet`).  When I run `dotnet --info` it now says version 1.0.1.  I think installing the latest x86 binaries would have worked too.
